README.md

```diff
@@ -39,18 +39,34 @@ Then you can run the program using
 
 `.\geneva-cli.exe intercept --interface <interface-name> -strategyFile .\s.txt`
 
+or
+
+`.\geneva-cli.exe intercept --interface <interface-name> -strategy "[TCP:flags:PA]-fragment{tcp:-1:True}-| \/"`
+
+[TCP:flags:PA]-fragment{tcp:8:True}(,fragment{tcp:4:True})-| \/
+
+
 You can find a list of available interfaces using
 
 `.\geneva-cli.exe list-adapters`
 
+### How to find strategies
+You can find a list of strategies in strategies.txt or in the [Geneva paper](http://geneva.cs.umd.edu/papers/geneva_ccs19.pdf) on page 9 with tested success rates for China, India, and Kazakhstan.
+
 ## How to test
 
-You can test or rather verify that the intercept mode is running by simply running it with a validated strategy. The output should show that packets are being rerouted. You can monitor your adapter in [WireShark](https://www.wireshark.org/) and you should notice a huge uptick in packets being sent from your adapter.
+You can test or rather verify that the intercept mode is running by simply running it with a validated strategy. The output should show that packets are being rerouted. You can monitor your adapter in [WireShark](https://www.Wireshark.org/) and you should notice a huge uptick in packets being sent from your adapter.
```

Comment:
```suggestion
You can test or rather verify that the intercept mode is running by simply running it with a validated strategy. The output should show that packets are being rerouted. You can monitor your adapter in [WireShark](https://www.wireshark.org/) and you should notice a huge uptick in packets being sent from your adapter.
```

Reply:
So funnily this happened because I did a find and replace to capitalize the 'W' elsewhere